scripts/generate-examples.py

```diff
@@ -111,8 +111,10 @@
                 options.update(metaOptions)
 
             # If the example has additional options, load them.
-            example_options: str = example.get("options", "{}")
-            options.update(json.loads(example_options))
+            example_options: str = example.get("options", {})
+            if example_options:
```

Comment:
You don't really need the 'if' -- the second variable in the `.get` is what it should return if there are no options. By default this is `None`, but you've specified an empty dictionary. Doing an update with an empty dictionary will always succeed, so you can run it either way and it will do the right thing.

Reply:
The `if` was only to have a log in that case